pyproject.toml

```diff
@@ -72,4 +72,15 @@ requires = [
 build-backend = "poetry.core.masonry.api"
 
 [tool.mypy]
-ignore_missing_imports = true
\ No newline at end of file
+ignore_missing_imports = true
+
+[tool.pytest.ini_options]
+asyncio_mode = "auto"
+# NOTE: changing default location of pytest_cache because the uvicorn file watcher somehow triggered too many reloads
+cache_dir = "../.pytest_cache" 
+testpaths = [
+    "tests"
+]
+
+[tool.poetry.plugins.pytest11]
```

Comment:
What the purpose of this block?

Reply:
It indicates the start of settings related to pytest plugin. pytest11 refers to the plugin architecture. By offering a `pytest-rosys` plugin, other sub-projects can use the fixtures from rosys/testing/fixtures.py.